setup.py

```diff
@@ -1,58 +0,0 @@
-from setuptools import setup, find_packages
-from codecs import open
-from os import path
-
-import spinegeneric
-
-# Get the directory where this current file is saved
-here = path.abspath(path.dirname(__file__))
-
-with open(path.join(here, "README.md"), encoding="utf-8") as f:
-    long_description = f.read()
-
-req_path = path.join(here, "requirements.txt")
-with open(req_path, "r") as f:
-    install_reqs = f.read().strip()
-    install_reqs = install_reqs.split("\n")
-
-setup(
-    name="spinegeneric",
-    version=spinegeneric.__version__,
-    python_requires=">=3.9",
```

Comment:
Apologies if I'm being dense here, but if the old `setup.py` supported Python 3.9, in what way is this PR upgrading the Python version?

Reply:
You're absolutely right, it's not! I think I got confused because the Github Actions for this repo, and for data-single-subject, and for data-multi-subject, all pin different releases of the repo. And at least one of them used to fail because Python 3.7 was no longer supported. I'll update the PR title.